core/trie/key.go

```diff
@@ -0,0 +1,119 @@
+package trie
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"math/big"
+
+	"github.com/NethermindEth/juno/core/felt"
+)
+
+type Key struct {
+	length uint8
```

Comment:
nit: lets change it to `len` as it a go idiom

Reply:
Done :+1: